components/inputs/demo/input-date.html

```diff
@@ -41,6 +41,13 @@ <h2>Disabled</h2>
 					<d2l-input-date label="Name (disabled)" disabled></d2l-input-date>
 				</template>
 			</d2l-demo-snippet>
+
+			<h2>Empty State Text</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-input-date label="Graduation Date" empty-text="No graduation date"></d2l-input-date>
```

Comment:
Are we sure that we need to expose `empty-text` on the API, and that we can't use some generic term.  If we can use a generic term, then the max-width stuff might be simplified.

Nit: the heading for the demo says "Empty State Text" but the attribute is `empty-text`.  Looks like there are a couple other references to "state" in the code.  May want to be consistent in naming.

Reply:
We do want to expose `empty-text` - it seems the value will likely vary depending on context/usage.

Reply:
Plus different languages and the user's font size & zoom settings would impact its size, so I think we need to calculate no matter what.